README.md

```diff
@@ -1,8 +1,18 @@
 [![Build and Test](https://github.com/dafny-lang/dafny/workflows/Build%20and%20Test/badge.svg)](https://github.com/dafny-lang/dafny/actions?query=workflow%3A%22Build+and+Test%22) [![Gitter](https://badges.gitter.im/dafny-lang/community.svg)](https://gitter.im/dafny-lang/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
+Dafny is a **programming language teamed up with a verifier**. As you type in your program, the verifier constantly looks over your shoulders, flags any errors, shows you counterexamples, and congratulates you when your code matches your specifications. When you're done, Dafny **compiles your code to C#, Java, JavaScript and Go**, so it can integrate with your existing workflow.
```

Comment:
This is a better wording than (what had been on L5) previously. But it sells Dafny a bit short. It sounds like you could just add a verifier to an existing language and get the goodness of Dafny. You cannot, because you need specifications, the ability to write proofs, and a language semantics that you stand a chance of verifying. Would "Dafny is a programming language whose design and tool set are aimed at doing verification" sound too bulky?

Reply:
Why do we not like the former "verification-aware", or "verification-ready" programming language?

Reply:
I hear what you say, and that's why I try to put in the simplest words possible there.
Note that you *can* add a verifier to an existing language and get a Dafny-like goodness, provided that this language is rich enough. This is what Stainless from EPFL achieved with Scala: No new constructs, just the language itself + verification on top of it (through reading the AST), the ability to write proofs, and the language semantics stand a chance of verifying.
But the difference is that the verified is not "teamed up" or "integrated" into the language, and that's what I want to say here. Two things can't team up if they don't work together.
The next paragraph highlights the difference with what you mention, which is that specifications are written like the code.

> Dafny is a programming language whose design and tool set are aimed at doing verification

Verification is not the aim (software developers do no want to "do verification" per se, only implement features that will make their customers happy). Verification is a **killer feature**, but the aim is to help developers write correct code. I think this sentence, is not appropriate in this context.

Reply:
I like "verification-aware" and "verification-ready" a lot. I even prefer the latter, because it's not just that Dafny is "aware" that it might be verified, as @RustanLeino  noted it's designed for that purpose.